pkg/flowcli/project/keys.go

```diff
@@ -40,6 +40,7 @@ type AccountKey interface {
 	Signer(ctx context.Context) (crypto.Signer, error)
 	ToConfig() config.AccountKey
 	Validate() error
+	PrivateKey() (*crypto.PrivateKey, error)
```

Comment:
I like this since there is a lot of places that need to access the private key, and doing `ToConfig` and then accessing the private key is very awkward not to mention it doesn't work as it should since KMS can't provide this property, so I think abstracting getter for the `PrivateKey()` can be good as the retrieval is not always just get the property from config but can include other ways (think KMS (not allowed), or wallet signing etc...)